pkg/consts/consts.go

```diff
@@ -11,6 +11,14 @@ const (
 	// tx inclusion proof via the ABCI Query method. The desired transaction
 	// index must be formatted into the path.
 	TxInclusionProofQueryPath = "custom/txInclusionProof/%d"
+
+	// ProtoBlobTxTypeID is included in each encoded BlobTx to help prevent
+	// decoding binaries that are not actually BlobTxs.
+	ProtoBlobTxTypeID = "BLOB"
+
+	// ProtoBlobTxTypeID is included in each encoded MalleatedTx to help prevent
+	// decoding binaries that are not actually MalleatedTxs.
+	ProtoMalleatedTxTypeID = "MLTD"
```

Comment:
yeah, I stuck with 4 letters to stay consistent, and I feel like that adds enough accidental collision protection. It can still be pretty easily faked tho if someone wants to do it.